examples/h2-client/h2_client.bal

```diff
@@ -1,7 +1,7 @@
 import ballerina/h2;
 import ballerina/io;
 
-// Create a client for h2 database. Change the path before running the sample.
+// Create a `client` for h2 database. Change the path before running the sample.
```

Comment:
Shall we reword this to Create a client endpoint for h2 database. IMO highlighting is not required as "client" is not a keyword

Reply:
"client" is a keyword. h2:Client is a `client` type object. Should we still add "endpoint"?

Reply:
I was thinking since `client` object is not visible in the example as a keyword. (But the h2:Client) What we have is two types of endpoints - client endpoints and listener endpoints. So IMO it is better if we can have more descriptive word. But it is fine. :)

Reply:
Yeah, thats true. .Updated the examples